	No need to directly #include <kpathsea/win32lib.h> (already done
	from <kpathsea/config.h>).

diff -ur cjkutils-4.8.2.orig/cjklatex/cjklatex.c cjkutils-4.8.2/cjklatex/cjklatex.c
--- cjkutils-4.8.2.orig/cjklatex/cjklatex.c	2010-12-19 16:25:43.000000000 +0100
+++ cjkutils-4.8.2/cjklatex/cjklatex.c	2011-09-12 13:44:29.000000000 +0200
@@ -36,10 +36,6 @@
 #include <kpathsea/lib.h>
 #include <kpathsea/getopt.h>
 
-#if defined(WIN32) && !defined(__MINGW32__)
-#include <kpathsea/win32lib.h>
-#endif
-
 static const char *cjklatex_version_string = "1.0";
 
 static const char *usage_str[] = {
